moped-editor/src/views/projects/projectView/ProjectPhaseDateConfirmationPopover.js

```diff
@@ -35,7 +35,7 @@ const DateConfirmationPopover = ({ children, isEnabled, dateType }) => {
         anchorEl={anchorEl}
         anchorOrigin={{
           vertical: "center",
-          horizontal: "right",
+          horizontal: "center",
```

Comment:
Something about how datagrid pro handles its spacing, this popover was appearing too far away from the anchor.